multipart/src/main.rs

```diff
@@ -80,18 +49,19 @@ fn index() -> HttpResponse {
 
 fn main() -> std::io::Result<()> {
     std::env::set_var("RUST_LOG", "actix_server=info,actix_web=info");
-    env_logger::init();
-
+    std::fs::create_dir_all("./tmp").unwrap();
+    let config = envy::from_env::<Config>().unwrap();
+    let p =  config.port;
+    let port = concat_string!("0.0.0.0:", p);
```

Comment:
This isn't a port, it includes the ip address, so the naming is misleading.

I think the configurable port is a distraction here. Only one other example in this repository has a configurable port and it's https://github.com/actix/examples/blob/dc6e1cc7625179e5ab8c037d20ce6883e3f77f93/http-proxy/src/main.rs#L84 which makes sense given it's a demonstration of an http proxy.

Reply:
I want the example to be deployable to Heroku. Herok sets the ENV port.

Reply:
The IP address is preset to 0.0.0.0 so it is only setting the port.

Reply:
>I want the example to be deployable to Heroku. Heroku sets the ENV port.

@z0mbie42 okay, that's an understandable desire but what does Heroku have to do with this particular example or this repository? None of the other examples in this repository respect the port environment variable that Heroku obligates you to use.

Reply:
Well, I wanted to have my own repo and just put it in the readme - but they wanted the code in this repo. So here we are. Examples aren't useful unless you can deploy them. My example is a 2.0 example anyway. So probably they should be all changed. But for now, I can hard code it and do another PR later. All the examples will have to be updated to 2.0 anyway later.

Reply:
Thanks for the great feedback!